db/migrate/20240617140927_change_columnto_use_sequence.rb

```diff
@@ -0,0 +1,4 @@
+class ChangeColumntoUseSequence < ActiveRecord::Migration[6.1]
+  def change
+  end
+end
```

Comment:
is this for the sequence changes already?

Reply:
Yes it is the preparation for it so we can merge it with the next PR directly into our psql-migration branch